paper.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'GCM-Filters: A Python package for Diffusion-based Spatial Filtering of Gridded Data from General Circulation Models'
+tags:
+  - Python
+  - ocean modeling
+  - climate modeling
+  - fluid dynamics
+authors:
+  - name: TBD
+    affiliation: 1
+affiliations:
+  - name: TBD
+    index: 1
+date: 16 June 2021
+bibliography: paper.bib
+
+---
+
+# Summary
+
+`GCM-Filters` is a python package that allows scientists to perform spatial filtering analysis in an easy, flexible and efficient way. The package implements the filtering method that was introduced by @grooms2021diffusion. The filtering algorithm is analogous to smoothing via diffusion; hence the name *diffusion-based filters*. `GCM-Filters` is designed to work with gridded data that is produced by General Circulation Models (GCMs) of ocean, weather, and climate. Spatial filtering of GCM data is a common analysis method in the Earth Sciences, for example to study oceanic and atmospheric motions at different spatial scales or to develop subgrid-scale parameterizations for ocean models.
```

Comment:
All examples are using ocean model outputs. So, I was not sure if the current version of GCM-Filters could handle atmospheric model outputs, which are generally provided on an equidistant lat-lon grid? Do atmospheric models use the spherical coordinate system or rectangular coordinate system? If it is former, that would require applying Laplacian in spherical coordinates? Does GCM-Filters have an option for that?

Reply:
Gcm-filters should work great with finite-volume atmospheric models like [FV3](https://www.gfdl.noaa.gov/fv3/), since their Laplacians are very similar to ocean models. The caveat is that we would need to implement the correct boundary conditions for the cubed-sphere grid.

Many atmospheric models use spectral methods and gaussian grids. For these, we would need to implement the appropriate Laplacian in a different way. [Windspharm](https://ajdawson.github.io/windspharm/latest/) might have some routines that could be helpful here.

Finally, there are models like [MPAS](https://mpas-dev.github.io/atmosphere/atmosphere.html) which use unstructured hexagonal meshes. These have a different Laplacian as well.

My hope is that we can keep working on gcm-filters and eventually implement all of these cases.

Reply:
@rabernat Thanks. So, GCM-Filters should work for atmospheric models, but it requires implementing the correct Laplacian form and boundary conditions. 

I observed that implemented Laplacian forms are mentioned in lines 55-63. Maybe after this, it could be mentioned that atmospheric model outputs are not yet supported, but can be implemented within the GCM-Filters framework.

Reply:
Thanks for this comment, @hmkhatri! I have included a note to this effect in the newest version of the paper.